tests/test_errors.py

```diff
@@ -157,3 +157,14 @@ def test_t1_negative_output_dims() -> None:
         ValueError, match="Got output_shape that was not positive integer"
     ):
         pytorch_finufft.functional.finufft_type1.apply(points, values, (10, -2))
+
+# dependencies
+def test_cuda_finufft_not_installed():
+    pytest.importorskip("finufft")
```

Comment:
Is this == "if we can import finufft, import it, otherwise skip this test"?

Reply:
Yes, which I think means it’s actually the flip of what I wanted it to be, which is a mistake (which I think also lead to the confusion in your second comment, and the test to fail on Jenkins)

I’ll fix it up tomorrow if I can, otherwise this will just be untested which I think is fine